scripts/migrate-audit-enabled-default.sh

```
@@ -0,0 +1,74 @@
+    if ! grep -q '\${REPLICATION_BACKEND_AUDIT_ENABLED:-false}' "$compose_file"; then
+        echo "[$instance] docker-compose.yml already up to date or pattern not found, skipping"
+        return
+    fi
+
+    echo "[$instance] migrating..."
+
+    # Backup before editing
+    backupFile "$compose_file"
+
+    # Replace the default from false to true
+    sed -i 's/\${REPLICATION_BACKEND_AUDIT_ENABLED:-false}/\${REPLICATION_BACKEND_AUDIT_ENABLED:-true}/g' "$compose_file"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)